packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -174842,5 +174852,986 @@
         ]
       }
     ]
-  }
+  },
+  "monomerTemplate-Biotin___Biotin": {
```

Comment:
You can shorten this key as well as `id` property within as it's built as `${alias}___${fullName}`, thus it has to be `monomerTemplate-B7___Biotin`
The same is for the other CHEMs you've added